### DISCUSSION TITLE
Support asciinema

### DISCUSSION THREAD

**Discussion by ystxn** [GitbookIO/community#258] [Feature Requests]
Allow embedding of asciinema recordings of console activity for documentation sharing purposes. Much higher quality, storage / bandwidth efficient and easier than having to record videos and uploading them onto gitbook.

**Comment by addisonschultz** [comment]
Great feature request - I actually think this might be able to be supported with our Integration Platform 🤔 

We use https://iframely.com/domains under the hood to embed links in our editor - but with our platform you'll have more control on unfurling links in the editor - we even have a guide on that here: https://developer.gitbook.com/getting-started/guides/create-a-custom-unfurl-action-for-your-integration

Would be super cool to see if someone put an integration together for this!
